launch_ros/launch_ros/actions/load_composable_nodes.py

```diff
@@ -220,6 +220,13 @@ def execute(
                 'target container is neither a ComposableNodeContainer nor a SubstitutionType')
             return
 
+        # Check if a global ros_namespace has been set with push ros namespace.
+        base_ns = context.launch_configurations.get('ros_namespace', None)
+        if base_ns is not None:
+            self.__final_target_container_name = make_namespace_absolute(
```

Comment:
this will be applied to the service client to load the composable nodes below.
without having this fix, service path is not appended with namespace, that fails to call the load request for target container.